entrypoint.sh

```diff
@@ -23,6 +23,20 @@ if [ -z "$DD_SERVICE" ]; then
     exit 1
 fi
 
+########################################################
+# osv-scanner
+########################################################
+mkdir /osv-scanner
+if [ "$(uname -m)" = "aarch64" ]; then
+  echo "Installing osv-scanner for ARM64"
+  curl -L -o "/osv-scanner/osv-scanner.zip" "https://github.com/DataDog/osv-scanner/releases/download/v0.7.1/osv-scanner_linux_arm64.zip" >/dev/null 2>&1 || exit 1
```

Comment:
Was it intended to have a version bump here as well? I don't see it in the description

Reply:
yes